packages/web/src/scss/components/Item/README.md

```diff
@@ -1,10 +1,88 @@
 # Item
 
-The Item component is used to display a single item in a list. Currently we support
-two types of item content: Radio and Checkbox. In order to unify and simplify
-API of these components and also to avoid repeating ourselves, we use Item as their modifier.
+The Item component is used to display a single item in a list. It can be used in Dropdown or similar.
+To use Item with checkbox or radio please use components [Checkbox][checkbox] or [Radio][radio]
+with `item` modifier. We do this to avoid repeating the same code and to simplify the API.
 
-So, to create an Item of Radio content, you need to add `Radio--item` modifier class.
+Simple Item example:
+
+```html
+<div class="Item">
```

Comment:
Any explanation why it's just a `<div>`? The more I think about it, `<a>` seems as a better default value to me.

Reply:
I decided for `button` as this will probably the most common use case with `a`